package.json

```diff
@@ -37,7 +37,6 @@
     "ember-cli-uglify": "^1.2.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-getowner-polyfill": "1.0.0",
```

Comment:
Removing this will actually break our support for versions < Ember 2.3. We should still include the getowner polyfill, but not import it in the modules where we use it to resolve the deprecation.